py_gnome/gnome/outputters/outputter.py

```diff
@@ -310,8 +311,12 @@ def write_output(self, step_num, islast_step=False):
             else:
                 self._write_step = False
 
+
         if (islast_step and self.output_last_step):
             self._write_step = True
+        elif islast_step:
+            self._write_step = False
```

Comment:
I understand what you are going for with `output_last_step`.

I wanted to use the option and set it to `False` because interpolating to a surface/raster falls apart when the individual splots are spread widely. When I tried to set `output_last_step` to False in the rasterer outputter, it was still outputting the last time step. So I figured there has to be something wrong with the logic associated with the last step option. I might have "fixed" the wrong thing though.

Reply:
ahhh -- got it. But if the algorithm falls apart under some conditions, that needs to be caught elsewhere -- it happening at the the last time step would be a coincidence.

But we should document what "ouput_last_timestep" does better.